test/test_proxy_tensor.py

```diff
@@ -1008,7 +1008,8 @@ def f(x, y):
         self.assertExpectedInline(r, """\
 def forward(self, x_1, y_1):
     sym_size_int = torch.ops.aten.sym_size.int(y_1, 0);  y_1 = None
-    resize_ = torch.ops.aten.resize_.default(x_1, [sym_size_int]);  x_1 = sym_size_int = None
+    resize_ = torch.ops.aten.resize_.default(x_1, [sym_size_int]);  x_1 = None
+    eq = 0 == sym_size_int;  sym_size_int = None
```

Comment:
In general, "stray" new SymInt computation that show up in the graph are because of operators that are traced in a composite way before they hit their real payload. For example, I am pretty sure this zero test happens outside of the resize_ implementation in C++, so it ends up being traced.